docs/src/userguide/runningcode.md

```diff
@@ -1 +1,59 @@
+- `args`: an array of command line arguments. The values specified in this list will be passed as command line arguments to the Julia process that is started and are available via the `Base.ARGS` variable in the Julia script. If this attribute is not specified, no command line arguments are passed to the Julia instance.
+
+## The Julia REPL
+
+The Julia extension provides a Julia REPL inside VS Code. You can start this REPL with the `Julia: Start REPL` command.
+
+The Julia REPL in the extension is identical to the default Julia REPL, but adds a number of additional integrations (plot pane, grid viewer, debugger etc.) that are not part of the standard REPL. Note that only the REPL that you start with the `Julia: Start REPL` command has these integrations. If you start Julia from a system shell inside VS Code, it won't provide these integration points.
+
+The REPL that is started with the `Julia: Start REPL` command will have the root folder of the currently active workspace as its working directory, and will be started with the Julia project that is currently active in the VS Code window.
+
+## Running code in the Julia REPL
+
+There are four commands that you can use to run code from your editor in the Julia REPL: `Julia: Execute Code` (Ctrl+Enter), `Julia: Execute Code Block` (Alt+Enter), `Julia: Execute Code Cell` (Shift+Enter) and `Julia: Execute File`.
+
+### Julia: Execute Code (Ctrl+Enter)
+
+This command will either send the text that is currently selected in the active editor to the Julia REPL, or it will send the entire line where the cursor is currently positioned when no text is selected. In the latter case it will also advance the cursor to the next line of code.
+
+This command works a little bit like copy-paste: it will literally just copy some code from the editor into the REPL as if that code was typed in the REPL. This design has some consequences that you should keep in mind:
+- when you send something like a function definition to the REPL, no line or file information will be associated with that function. This can be annoying later on, when exceptions that are thrown in this function will not show any location information.
+- `include` statements will always try to load relative paths relative to the current working directory, not relative to the file from which they are executed.
+- Macros like `@__DIR__` and `@__FILE__` do not work properly.
+
+Due to these limitations, most users should use the `Julia: Execute Code Block` command in most cases.
+
+### Julia: Execute Code Block (Alt+Enter)
+
+Whenever there is some Julia code selected in the currently active editor, this command will execute the selected code. If no text is selected, the command will identify the smallest self contained code block surround the current cursor position and execute that code block.
```

Comment:
for `smallest self contained code block` perhaps substitute `largest self contained code block that is not a module definition`?

Reply:
Or maybe something like `the command will identify the extent of the top-level language construct that the cursor is located in (except modules) and execute that code block`?

Reply:
Yep that works!